packages/config/src/layer2s/index.test.ts

```diff
@@ -484,6 +484,19 @@ describe('layer2s', () => {
       }
     })
   })
+
+  describe('state validation', () => {
+    describe('every description ends with a dot', () => {
+      for (const layer2 of layer2s) {
+        if (!layer2.stateValidation) continue
+
+        expect(layer2.stateValidation?.description.endsWith('.')).toEqual(true)
```

Comment:
Is there a need for optional operator?

Reply:
yup, we will not specify state validation for every project